kolibri/content/utils/paths.py

```diff
@@ -1,13 +1,22 @@
 import os
+import re
 
 from django.conf import settings
+from django.core.urlresolvers import reverse
 
 try:
     from urlparse import urljoin
 except ImportError:
     from urllib.parse import urljoin
 
 
+# valid storage filenames consist of 32-char hex plus a file extension
+VALID_STORAGE_FILENAME = re.compile("[0-9a-f]{32}\.[0-9a-z]+")
```

Comment:
Mostly worried about lack of DRY -- spoke with @aronasorman about the plans he has for extracting shared constants/utils into a separate library, and that this could later be specified in there and used in both the CC server and Kolibri. In the meantime, I don't think it would buy us much. The list of supported extensions is also contingent upon the currently activated plugins, so hardcoding them all here might not be ideal. Listing the zip extensions here was already a bit of a pragmatic short-term compromise.